galera/orchestrate/master.sls

```diff
@@ -0,0 +1,16 @@
+galera.master:
+  salt.state:
+    - tgt: 'galera:master'
+    - tgt_type: pillar
+    - queue: True
```

Comment:
I would add `batch: 1` here just to be sure (eg. in case of future multi-master setup), etc.